package.json

```diff
@@ -74,8 +75,15 @@
     "sinon": "^9.0.3",
     "snyk": "^1.381.1"
   },
-  "optionalDependencies": {},
+  "overrides": {
+    "connect-multiparty@^2.2.0": {
+      "qs": "^6.5.3"
+    },
+    "request@^2.88.0": {
```

Comment:
We should consider getting rid of `request`, any thoughts on how big of a lift this would be?